eng/tools/generator/cmd/issue/request/request.go

```diff
@@ -103,11 +106,22 @@ func NewReleaseRequestIssue(issue github.Issue) (*ReleaseRequestIssue, error) {
 		linkKeyword, tagKeyword, releaseDateKeyword,
 	})
 
+	// get release date
+	targetDate := regexp.MustCompile(`\d*-\d*-\d*`).FindString(contents[releaseDateKeyword])
+	releaseDate, err := time.Parse("2006-01-02", targetDate)
+	if err != nil {
```

Comment:
I'd prefer to use today as `targetDate` than throw an error.

Reply:
fix